packages/react-core/src/components/NumberInput/examples/WithStatus.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { NumberInput, ValidatedOptions } from '@patternfly/react-core';
+
+export const WithStatus: React.FunctionComponent = () => {
+  const max = 10;
+  const min = 0;
+
+  const [validated, setValidated] = React.useState<ValidatedOptions>(ValidatedOptions.success);
```

Comment:
Since you are using this `ValidatedOptions` examples, I would add it to the prop table (line 5 of the md file, `propComponents`) so consumers can easily see the definition.

Reply:
I think `propComponents` is reserved for components while ValidatedOptions is just an enum. We could add it as the type for the `validated` property within NumberInput.. but I remember not doing that in the past for the reason of the parser that creates the documentation not being able to drill down. So within the NumberInput props, we'd see the validated prop like this:
![image](https://user-images.githubusercontent.com/5322142/185422974-90293d04-12b4-424d-afb1-5b7e953b402c.png)

Reply:
or if we combine options... 
![image](https://user-images.githubusercontent.com/5322142/185424374-35207dd7-1630-480b-9897-9393016ba1f7.png)

Reply:
@gitdallas Yes you are right.   And that is why for our props we list about the values, for documentation.  I think your second screenshot here would be good.